docs/book/src/cronjob-tutorial/testdata/project/cmd/main.go

```diff
@@ -72,11 +74,17 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var secureMetrics bool
+	var enableHTTP2 bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&secureMetrics, "metrics-secure", false,
+		"Whether or not the metrics endpoint should be served securely")
```

Comment:
Could we try to following the standard of the other commets
Something more like 

If set enable the ...

Reply:
Addressed in https://github.com/kubernetes-sigs/kubebuilder/pull/3734/commits/2375fa34d34e520eded52b4ac0e1c63795d9a5a9